bse-api-steps/src/main/java/com/quantori/bse/qa/rest/steps/library/LibrariesStep.java

```diff
@@ -2,32 +2,79 @@
 
 import io.qameta.allure.Step;
 import lombok.extern.slf4j.Slf4j;
-import com.quantori.bse.qa.rest.Services;
-import com.quantori.bse.qa.rest.assertions.CommonResponseAssert;
+import retrofit2.Call;
 import com.quantori.bse.qa.rest.enums.LibraryType;
 import com.quantori.bse.qa.rest.enums.Storage;
 import com.quantori.bse.qa.rest.models.Library;
+import com.quantori.bse.qa.rest.models.OutputBoolean;
+import com.quantori.bse.qa.rest.models.ResponseLibrary;
 import com.quantori.bse.qa.rest.models.ServiceData;
+import com.quantori.bse.qa.rest.services.AbstractServices;
+import com.quantori.bse.qa.rest.services.LibraryControllerService;
+
+import java.util.List;
 
 @Slf4j
 public class LibrariesStep {
 
-    @Step("Create library {serviceData.name} in storage {storage.value} ")
-    public static void createLibrary(ServiceData serviceData, Storage storage) {
-        CommonResponseAssert.assertResponse(
-                () -> Services.libraryControllerService().postLibrariesStorage(storage.value, serviceData));
+    @Step("create library {serviceData.name} in storage {storage.value} ")
+    public static Call<OutputBoolean> createLibrary(Storage storage, ServiceData library) {
+        return createLibrary(AbstractServices.libraryControllerService(), storage, library);
```

Comment:
там инициализация идёт в тестах. Так тоже должно сработать, но всё же лучше перестраховаться
```suggestion
        return createLibrary(Services.libraryControllerService(), storage, library);
```